lib/bracket_attributes.ex

```diff
@@ -33,11 +33,29 @@ defmodule BracketAttributes do
     |> concat(choice([word(), double_quoted_string()]))
     |> wrap()
 
+  defparsec(
+    :nested_attribute,
+    choice([
+      string("attr")
+      |> wrap(parsec(:maybe_brackets))
+      |> post_traverse({:flip_attr, []})
+      |> wrap(),
+      word()
+      |> wrap(parsec(:maybe_brackets))
+      |> parsec(:content_name)
+      |> wrap()
+    ]),
+    export_combinator: true
+  )
+
   defparsec(
     :maybe_brackets,
-    ignore(string("("))
+    ignore_whitespace()
+    |> ignore(string("("))
+    |> ignore_whitespace()
     |> repeat(
       choice([
+        parsec(:nested_attribute),
```

Comment:
Test for nested attributes early.

When parsing `bar(baz(foo))` we should check if baz is an attribute before checking if it is a word.